packages/web-components/src/components/cbp-segmented-button-group/cbp-segmented-button-group.stories.tsx

```diff
@@ -23,8 +23,8 @@ export default {
 };
 
 function generateButtons(buttons) {
-  const html = buttons.map(({ label, value, pressed, disabled }) => {
-    return `<cbp-button type="button" value="${value}" ${pressed == true ? 'pressed' : ''} ${disabled == true ? 'disabled' : ''}>${label}</cbp-button>`;
+  const html = buttons.map(({ label, value, pressed, disabled, variant }) => {
+    return `<cbp-button type="button" value="${value}" ${pressed == true ? 'pressed' : ''} ${disabled == true ? 'disabled' : ''} variant="${variant ? variant : 'default'}">${label}</cbp-button>`;
```

Comment:
I don't think we want `variant="default"` output in code to copy-paste. Just output an empty string if it's not specified (like in the first story).

Reply:
updated